pkg/v1/remote/catalog_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package remote
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-containerregistry/pkg/name"
+)
+
+func TestGetCatalogPage(t *testing.T) {
+	cases := []struct {
+		name         string
+		responseBody []byte
+		wantErr      bool
+		wantRepos    []string
+	}{{
+		name:         "success",
+		responseBody: []byte(`{"repositories":["test/test","foo/bar"]}`),
+		wantErr:      false,
+		wantRepos:    []string{"test/test", "foo/bar"},
+	}, {
+		name:         "not json",
+		responseBody: []byte("notjson"),
+		wantErr:      true,
+	}}
+	//TODO: add test cases for pagination
```

Comment:
Planning to do this later? :smile: 

supernit: add a space between `//` and `TODO`

Reply:
Potentially, but I just wanted to note that they are missing if anyone runs into issues there later and wants to write some.
Do you need this?